src/components/Navigation.js

```diff
@@ -1,18 +1,26 @@
-import React from 'react';
+import React from "react";
+import { Link } from "react-router-dom";
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
+
         <div>
-          <a href="">Home</a>
+          <Link to="/">
+            <a href="">Home</a>
```

Comment:
this 'Link' component is a modified a tag from react-router-dom, so we dont need another a tag inside it
```suggestion
            Home
```